### PR TITLE
Add namespace to functions

### PR DESCRIPTION
**Pull request by szepeviktor**
Namespace is a PHP 5.3 feature.
https://www.php.net/manual/en/language.namespaces.rationale.php

You may be interested in leaving legacy technologies behind: https://github.com/szepeviktor/small-project/blob/master/MAIN-FILE-PARTS.md

### CODE REVIEW

File: iconic-tpd.php [hunk 1]
```diff
@@ -9,6 +9,12 @@
  * Text Domain: iconic-tdp
  */
 
+namespace Iconic\Temporarily_Disable_Plugins;
+
+use function add_action;
+use function add_filter;
+// TODO Import all WordPress function.
```
Comment (szepeviktor) [inline iconic-tpd.php:16]: *s

Comment (jamesckemp) [reply iconic-tpd.php:16]: Should we import the rest - is it required?

Comment (szepeviktor) [reply iconic-tpd.php:16]: No it is not.
In a sense that new lines are not required in a PHP file.

Comment (szepeviktor) [reply iconic-tpd.php:16]: Custom namespaced functions may have the very same name as a WP core function.

Comment (jamesckemp) [reply iconic-tpd.php:16]: I suppose it isn't an issue because we don't have any functions with the same name, right?

Comment (szepeviktor) [reply iconic-tpd.php:16]: It is not, you're right.

**Pull request closed without merging**